docs/coalton-documentation-guide.md

```diff
@@ -33,6 +33,16 @@ that contains a filling of various sweet or savoury ingredients."
     (Meat String)  "A meat pie with the type of meat."))
 ```
 
+### `define-alias`
+
+`define-alias` allows for the same docstring style as `define`.
+
+```lisp
+(coalton-toplevel
+
+  (define-alias Index Integer
```

Comment:
In retrospect I honestly think we should have followed the pattern of

    (define <thing> <doc> <value>)

but you're right to follow existing behavior